spring-boot/proxy/src/main/java/eu/coatrack/proxy/security/LocalApiKeyVerifier.java

```diff
@@ -0,0 +1,85 @@
+package eu.coatrack.proxy.security;
+
+/*-
+ * #%L
+ * coatrack-proxy
+ * %%
+ * Copyright (C) 2013 - 2021 Corizon | Institut für angewandte Systemtechnik Bremen GmbH (ATB)
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import eu.coatrack.api.ApiKey;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+/**
+ * This class provides a method to determine an API keys validity.
+ *
+ * @author Christoph Baier
+ */
+
+@Service
+public class LocalApiKeyVerifier {
+
+    private static final Logger log = LoggerFactory.getLogger(ApiKeyAuthTokenVerifier.class);
+
+    private final LocalApiKeyManager localApiKeyManager;
+
+    public LocalApiKeyVerifier(LocalApiKeyManager localApiKeyManager) {
+        this.localApiKeyManager = localApiKeyManager;
+    }
+
+    public boolean isApiKeyAuthorizedConsideringLocalApiKeyList(String apiKeyValue) {
```

Comment:
It is hard for me to understand what this name should tell me. 

Maybe use something like `shouldGivenApiKeyBeAccepted`?

Reply:
Done.